source/_integrations/thread.markdown

```diff
@@ -322,6 +322,8 @@ You can only set a Thread network as preferred if the credentials are known.
 
    - Importing the credentials allows a Google- or Apple-created Thread network to be the preferred network of Home Assistant.
    <img width="400" src='/images/integrations/thread/thread-preferred-network.png'>
+3. If a **Make preferred network** button becomes available, select it.
```

Comment:
nit/optional: Just thought this might sound a bit more natural:
```suggestion
3. If the **Make preferred network** button is available, select it.
```

or even this:
```suggestion
3. If the **Make preferred network** button is shown, select it.
```